package.json

```diff
@@ -66,6 +65,7 @@
     "ember-cli-deploy-git-ci": "^1.0.1",
     "ember-cli-eslint": "^5.1.0",
     "ember-cli-inject-live-reload": "^2.0.1",
+    "ember-cli-sass": "^10.0.0",
```

Comment:
Is this still needed?

Reply:
Good catch - that may have been inadvertently reintroduced during my rebase this morning

Reply:
@mixonic I believe that's still in use by the ember-table docs - which is why it's moved to `devDependencies`

Reply:
Everything in `tests/dummy/app/styles` is still written in Sass

Reply:
In our fork, we replaced sass (scss) files with css and got rid of all the sass-related add-ons and scripts:
https://github.com/phorest/ember-table/commits/purge-sass

Would you consider accepting a PR to do that for this branch?

Reply:
Yeah I think so, if the change looks reasonable. It would only be a change to tests, as @kpfefferle mentioned the addon styles themselves have already been converted on this branch.

Reply:
This branch already removes the sass dependency from consuming apps which resolves the issues raised by users of the addon. I'm not sure we really have any compelling reason to remove it from the docs at this time 🤷‍♂️

Reply:
Seem ok to remove though, since the alternative is that we need to keep upgrading the dependency :-) Definitely not pressing.

Reply:
Sorry, missed it that ember-cli-sass is removed in this PR, carry on 👏🏼